kubernetes-manifests/kustomization.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+ - adservice.yaml
+ - cartservice.yaml
+ - checkoutservice.yaml
+ - currencyservice.yaml
+ - emailservice.yaml
+ - frontend.yaml
+ - loadgenerator.yaml
```

Comment:
```suggestion
# - loadgenerator.yaml # During development, the loadgenerator module inside skaffold.yaml will be used.
```

Reply:
**Context**
In https://github.com/GoogleCloudPlatform/microservices-demo/pull/609, we split the `skaffold.yaml` file into two modules: `app` and `loadgenerator`.
This means, today, we could run:
1. `skaffold run --default-repo=gcr.io/my-project --module=app` to develop without loadgenerator
2. `skaffold run --default-repo=gcr.io/my-project --module=app` to develop _with_ loadgenerator 
3. `skaffold run --default-repo=gcr.io/my-project` to develop _with_ loadgenerator  (identical to 2.)

The purpose of doing this was purely to showcase skaffold's "module" (or "multi-config") feature.

CC: @deesejohn (in case you had thoughts) :)

Reply:
**Thought:** 
We could also split the `loadgenerator` into its own module.
But we're have to make sure to include it by default inside [/kustomize/kustomization.yaml](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/kustomize/kustomization.yaml) because some tutorials/demo using Online Boutique prefer to have the `loadgenerator` create "traffic".

Reply:
I think leaving the `loadgenerator` module as `rawYaml` in the [skaffold.yaml](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/skaffold.yaml) is a useful example for showing off both renders (`rawYaml` and `kustomize`). I don't see any harm in splitting it out into its own component though.

Reply:
Thanks, @deesejohn, for your input. That helps (i.e., knowing that you're open to using either `rawYaml` or `kustomize` for the `loadgenerator`).